wt-status-serialize.c

```diff
@@ -78,10 +78,11 @@ static inline void wt_serialize_v1_changed(struct wt_status *s, int fd,
 	int len_path, len_rename_source;
 
 	trace_printf_key(&trace_serialize,
-		"change: %d %d %d %d %o %o %o %d %d %s %s '%s' '%s'",
+		"change: %d %d %d %d %d %o %o %o %d %d %s %s '%s' '%s'",
```

Comment:
Do we have to increase `v1` to `v2` so that existing serialized statuses will be ignored?

Reply:
i guess we should.  maybe.  technically, the v1 code has been in gvfs/master for months, but i don't think it has been used by anybody yet, because we're still waiting on gvfs server support, so i don't think it would hurt to leave it at v1 (this time).

Opinions??

Reply:
TODO Note to self.  Verify if the design doc needs updating.

Reply:
technically, yes. However, It has not been advertised, and none of our features are using this, so my opinion is to not bump the version number (and treat this feature as something we have not released yet).

Reply:
i'm going to keep the version number at 1 and go ahead and merge this.